std/array.d

```diff
@@ -380,6 +380,95 @@ if (isNarrowString!String)
     )));
 }
 
+/**
+ * Returns a static array from the stack with length of s and initializes
+ * it with copies of the elements of range $(D r). The remainder of the
+ * array is default-initialized.
```

Comment:
What happens if the range is longer than `s`?

Reply:
A standard overflow. But i think you meant that I add that in comment.